diy_asylum_frontend/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM node:8
+
```

Comment:
Is this version right? Current is 11.5 I thought

Reply:
Good call, just updated the version.